src/node/panner.rs

```diff
@@ -734,51 +759,30 @@ impl AudioProcessor for PannerRenderer {
                 projected_source = [0., 0., 1.];
             }
 
+            // Only handle mono for now (todo issue #44)
+            *output = input.clone();
+            let mut overall_gain_correction = 1.;
+            if output.number_of_channels() == 2 {
+                overall_gain_correction *= 2.; // stereo-to-stereo panning typically doubles volume
```

Comment:
```rs
overall_gain_correction = 2.; 
```

This is bit confusing and appear arbitrary, would be nice to have a bit more explanation